profiles/common/modules/features/events/events_resources/events_resources.module

```diff
@@ -145,10 +146,12 @@ function events_resources_fullcalendar_options_pre_view(&$settings, &$view) {
     $settings['fullcalendar']['header']['right'] .= ' resourceDay';
 
     // Get the values from the URL.
+    // @codingStandardsIgnoreStart: Extract will set $mode.
```

Comment:
Can't we simply exclude the check on unset variable, instead of excluding all CS checks?